bin/newman.js

```diff
@@ -34,7 +34,7 @@ cli(process.argv.slice(2), 'newman', function (err, args) {
     dispatch(args, function (err) {
         if (err) {
             console.error(err);
-            process.exit(1);
+            process.exit(_.toNumber(_.get(args, 'run.suppressExitCode', true)));
```

Comment:
Use a ternary operator in place of this is more suitable, because the behavior of _.isNumber is not documented if the argument is a boolean.
```
_.get(...) ? 1 : 0 
```
Also, should be false by default.